code/datums/components/defibrillator.dm

```diff
@@ -332,6 +340,28 @@
 	busy = FALSE
 	return
 
+/datum/component/defib/proc/combat_fibrillate(mob/user, mob/living/carbon/human/target)
+	if(!istype(target))
+		return
+	busy = TRUE
+	target.adjustStaminaLoss(60)
+	target.emote("gasp")
+	to_chat(target, "<span class='danger'>[user] touches [target] lightly with [parent]!</span>")
+	add_attack_logs(user, target, "Stunned with [parent]")
+	if(!(combat && target.body_position == LYING_DOWN && do_after(user, 1 SECONDS, TRUE, target, TRUE)))
+		target.KnockDown(4 SECONDS)
+		set_short_cooldown()
+		busy = FALSE
+		return
+	target.set_heartattack(TRUE)
+	target.visible_message("<span class='danger'>[user] has touched [target] with [parent]!</span>", \
+			"<span class='userdanger'>[user] touches you with [parent], and you feel a strong jolt!</span>")
+	playsound(get_turf(parent), 'sound/machines/defib_zap.ogg', 50, 1, -1)
+	SEND_SIGNAL(target, COMSIG_LIVING_MINOR_SHOCK, 100)
+	set_cooldown()
+	busy = FALSE
+	return
```

Comment:
```suggestion
```
implied return

Reply:
Lmao this is in the other one too, I'll remove both